packages/koa-shopify-webhooks/tsconfig.json

```diff
@@ -11,5 +11,5 @@
     "./src/**/*.tsx"
   ],
   "exclude": ["**/*.test.ts", "**/*.test.tsx"],
-  "references": [{"path": "../jest-koa-mocks"}, {"path": "../jest-dom-mocks"}]
+  "references": [{"path": "../network"}]
```

Comment:
* `jest-koa-mocks` is used in `packages/koa-shopify-webhooks/src/test/receive.test.ts`
* `jest-dom-mocks` is used in` packages/koa-shopify-webhooks/src/test/register.test.ts`

Reply:
Probably the second test is not correct. Will check :)

Reply:
Ok, I have pulled this locally and the references all seem to work so I'm OK with this.  These only just test files which we ignore during builds so that's fine.